R/dataone.R

```diff
@@ -51,10 +61,15 @@ get_token <- function(node) {
 
 #' Determine whether the set token is expired.
 #'
-#' @return
+#' @return (boolean)
```

Comment:
boolean is okay but, in R, booleans are of type `logical`, so I like to stick to that. Would you consider changing this to say `(logical)`?